helmfile/upstream/starboard-operator/Chart.yaml

```diff
@@ -10,6 +10,3 @@ sources:
 - https://github.com/aquasecurity/starboard
 type: application
 version: 0.9.1
-dependencies:
```

Comment:
can you add a note about this in this [README](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/upstream/README.md)
hopefully we will not overwrite this when we upgrade the upstream chart

Reply:
It was never part of the upstream. That is why i moved it.